src/components/change-requests/new-change-request/new-change-request-page/stage-gate-form-fileds/stage-gate-form-fields.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Form, FormControl, InputGroup, Row } from 'react-bootstrap';
+import { exampleAllUsers } from '../../../../../utils/lib/dummy-data';
```

Comment:
lint errors

Reply:
also be sure to not import things from `src/lib`. Anything coming from `utils` should be imported with an absolute package path of `'utils'`.